templates/kubeflow/features.html

```diff
@@ -160,7 +198,16 @@ <h2>Operational dashboards</h2>
         </div>
       </div>
       <div class="u-fixed-width">
-        <img src="https://assets.ubuntu.com/v1/9f6916dd-k8s-prometheus-light.png?w=1200&fmt=jpg" alt="" width="1040" />
+        {{
+          image(
+            url="https://assets.ubuntu.com/v1/9f6916dd-k8s-prometheus-light.png",
```

Comment:
Is there any benefit to converting this to a jpeg as per the original with the `fmt=jpg` query param?

Reply:
I didn't even see that, I'll re-add it